site/source/tutorials/internationalization.md

```diff
@@ -0,0 +1,504 @@
+---
+layout: false
+title: Internationalization
+overview: It is increasingly rare that applications reach a single audience that lives in a single region and speaks a single language. This article describes the tools Dojo 2 provides to guarantee an application can be adapted to all users, regardless of their language or address.
+---
+
+## Philosophy and Approach
```

Comment:
We've decided to lowercase follow-up words in the titles that are not proper nouns, so this should be Philosophy and approach (similar for other titles)